docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc

```diff
@@ -30,7 +30,7 @@ However, you need to move the configuration files into the correct location and
 
 [source, console]
 ----
-cp /opt/opennms/etc/examples/opennms.properties.d/timeseries.properties /opt/opennms/etc/opennms.properties.d/timeseries.properties
+cp /opt/opennms/etc/examples/opennms.properties.d/hosted_tsdb.timeseries.properties
```

Comment:
The command is invalid. There should be a source and destination file, but the change combined the two.